### ISSUE TITLE
Export Pre-trained model

### ISSUE THREAD

**Issue by xhj-fs** [UKPLab/sentence-transformers#60]
Hi, firstly thanks for the great work. May I know whether I could export pre-trained-models (use it in some of my existing code base) 

Thanks.

**Comment by nreimers** [comment]
Hi @xhj-fs 
sure, you can save the models on disc and also load them from there. You save them using the save method:
```
model.save('path/on/disc')
```

The individual components of the models are stored to separate folders and can be loaded individually.

**Comment by thesby** [comment]
@nreimers Can I save the model without parameter gradients?

**Comment by nreimers** [comment]
@thesby I think pytorch save method is just storing the parameters, not the gradients.

**Comment by thesby** [comment]
@nreimers  I find that the model is a pickle file. Is it possible to use other format to reduce the model file size?

**Comment by nreimers** [comment]
@thesby It uses torch save:
https://pytorch.org/docs/stable/generated/torch.save.html

Using another format would not be advisable as it would lead to many complications down the line. Also, reducing the file size is not really possible as you mainly store some (more-or-less) random floating point numbers.

Starting pytorch 1.6.0, the format is based on zip.

The only way to reduce the size would be to convert the floats first to FP16 before storing. I think calling ```model.half()``` before saving might do the trick.
